tests/test_search.py

```diff
@@ -129,33 +259,33 @@ def test_get_version_path_prefix_from_obj(
 @pytest.mark.fluent_version("latest")
 def test_search_from_root(capsys, new_watertight_workflow_session):
     meshing = new_watertight_workflow_session
-    pyfluent.search("display", search_root=meshing)
+    _search("display", search_root=meshing)
```

Comment:
I can't remember the discussion regarding the `search_root` argument. Did we consciously decide to remove it from the `search()` interface?

Reply:
We will include it new search function.

Reply:
Done. Thank you.